uint8 motor_num
---
TmcParam[] output
bool result